### ISSUE TITLE
[Sourcing][Accessibility] No aria-label and tooltip for breadcrumb

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9534]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
When the breadcrumb links are more , we get a 3 dots icon , blind users cannot understand what that icon button is for , so we need aria-label as well as tooltip for it.
this is reported by Deutsche telekom customer

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
core / platform: 0.39.1-rc-5

#### If this is a bug, please provide steps for reproducing it.
This is replicable on https://sap.github.io/fundamental-ngx/#/core/breadcrumb , try reducing screen size to see 3 dots.

the screenreader reads as ?? empty group right now : but instead it should read meaningful text and also as icon button (UI control type has to be read out)

<img width="1792" alt="Screenshot 2023-03-17 at 4 59 18 PM" src="https://user-images.githubusercontent.com/88182358/225892881-2e06ebf1-22a0-4182-9065-9236008c0e86.png">

**Comment by fkolar** [comment]
Confirmed. Missing aria-label. It should 
@droshev , @mikerodonnell89 

```html
<span...
role="presentation" 
 title="More"
 aria-label="More"
```

Plus I think wrapping element should have 

```html
<a ..
aria-expanded="false" | "true"
aria-haspopup="listbox" 
tabindex="0"

```

**Comment by fkolar** [comment]
PR created: 9610
